src/account_response.ts

```diff
@@ -83,4 +83,8 @@ export class AccountResponse {
     this._baseAccount.incrementSequenceNumber();
     this.sequence = this._baseAccount.sequenceNumber();
   }
+
+  public createSubaccount(id: string): MuxedAccount {
+    return this._baseAccount.createSubaccount(id);
+  }
```

Comment:
The `Account.createSubaccount()` method doesn't have a type annotation:

https://stellar.github.io/js-stellar-sdk/js-stellar-base_src_account.js.html

My guess is that TypeScript will assume `id` is of type `Any`, in which case `Account` would still be incompatible with `AccountResponse` because `createSubaccount()`'s function signature doesn't match.

Reply:
stellar-base is written in pure JavaScript, so its source will never have true type annotations (aside from in the jsdoc). They're provided after-the-fact in [index.d.ts](https://github.com/stellar/js-stellar-base/blob/master/types/index.d.ts#L13), where `id` is in fact specified to be a `string`. I've also made sure of this in a test script (based on #653).

**Aside:** The true long-term fix here is to make `Account` an `interface` just like in the Go SDK, with `AccountResponse` (at the SDK level) and another `Account` plus `MuxedAccount` (at the base level) being three implementations of it, but that's a bigger investment (and a major version bump) for Later:tm:.